Automattic/vip-go-mu-plugins
# Bump package-lock.json dependencies on vip-dashboard

## Description


Updating `package-lock.json` dependencies to address security issues. This has been done by removing the `package-lock.json` and then running `npm install` again.

Given that the current main dependencies in `package.json` are left untouched and they don't work with more recent versions of Node, we are also introducing a `.nvmrc` file to pin a Node version.

## Changelog Description

### VIP Dashboard dependency updates

Updates on underlying dependencies of VIP Dashboard to address potential security issues.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out this branch
2. Run npm install on vip-dashboard.
3. Build the JS changes.
4. Go to the VIP dashboard in wp-admin.
